scout/adapter/mongo/case.py

```diff
@@ -212,13 +212,11 @@ def _populate_name_query(self, query, name_query, owner=None, collaborator=None)
 
         if query_field == "user":
             LOG.debug(f"Search for cases with assignee '{query_term}'.")
-            users = self.user_collection.find({"name": {"$regex": query_term, "$options": "i"}})
-            nr_users = sum(
-                1
-                for i in self.user_collection.find(
-                    {"name": {"$regex": query_term, "$options": "i"}}
-                )
-            )
+            query_terms = query_term.split(" ")
+            user_query = {
+                "$or": [{"name": {"$regex": term, "$options": "i"}} for term in query_terms]
```

Comment:
👍

Reply:
I guess it will also behave a wee bit differently if you search for a specific Anna, but now it can find them - even if you get some non specific hits as well. 😸

Reply:
If you searched for "Anna" before and now you should get exactly the same results. Same if you searched by surname. What you get now if you search by "Name Surname" is All users where "Name" or "Surname" is in the name.

Reply:
And among these users there is also "Name Surname"

Reply:
Do we want to match exactly both terms? Then we substitute the "$OR" with the "$AND"

Reply:
Agreed, that’s how I read it. What someone might expect is only getting Name Surname, rather than all, but hey, also not complaining - this is better by a long shot than not getting a result at all!

Reply:
But then I fix it to get only Name Surname, since I'm already working on it!